### PR TITLE
[testing] Dashboard Update

### PR DESCRIPTION
**Pull request by AntoineYANG**
## Incoming Features
✅ Individual dashboard homepage, supporting two modes: `list` and `gallery`.
✅ Better edit view.
✅ Dashboard card with editable appearance, title and description.
✅ Highlight interaction on preview where dashboard data is read-only.
✅ Dashboard preview.
✅ Chart editor in dashboard.

## Future Work
🗓 Auto grid layout mode in editor.

### CODE REVIEW

File: packages/rath-client/src/pages/collection/index.tsx [hunk 1]
```diff
@@ -58,8 +78,75 @@ const Collection: React.FC = (props) => {
...
+                                    </PrimaryButton>
+                                </div>
+                            );
+                        }}
+                        isFooterAtBottom
+                    >
+                        <TextField
+                            label="title"
+                            value={configValue.title || ''}
+                            onChange={(e, data) => {
```
Comment (ObservedObserver) [inline packages/rath-client/src/pages/collection/index.tsx:116]: 抽一下

File: packages/rath-client/src/pages/dashboard/dashboard-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+                                card.content.text = d || undefined;
+                            })}
+                            multiline
+                            autoAdjustHeight
+                            resizable={false}
+                        />
+                        <ChoiceGroup
+                            label="Theme"
+                            selectedKey={card.config.appearance}
+                            options={CardThemes.map(thm => ({
```
Comment (ObservedObserver) [inline packages/rath-client/src/pages/dashboard/dashboard-panel.tsx:149]: 这个嵌套太变态了

File: packages/rath-client/src/pages/dashboard/edit-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+    return (
+        <Container>
+            <EncodingFieldContainer>
+                <Dropdown
+                    label="Mark"
+                    selectedKey={(mark as undefined | { type: typeof mark & string })?.type}
+                    onChange={(_, item) => card && item && runInAction(() => {
+                        const key = item.key as typeof MarkTypes[number];
+                        runInAction(() => {
```
Comment (ObservedObserver) [inline packages/rath-client/src/pages/dashboard/edit-panel.tsx:195]: 尽量避免store外部做操作

**Review by ObservedObserver — CHANGES_REQUESTED** [review]

File: packages/rath-client/src/pages/collection/index.tsx (hunk 1 above)
Comment (AntoineYANG) [reply packages/rath-client/src/pages/collection/index.tsx:116]: fix this @wangqimeng524163

**Pull request closed without merging**